readme.md

```diff
@@ -464,6 +465,20 @@ to confuse researchers.
 
 [R: minddisorders.com](http://www.minddisorders.com/Flu-Inv/Ganser-s-syndrome.html)
 
+#### [Multiple Personality Disorder](https://en.wikipedia.org/wiki/Dissociative_identity_disorder)
+
+> A disorder characterised by the presence of two or more distinct personality states
+
+Dissociative identity disorder (DID), also known as multiple personality disorder,
+is a mental disorder characterized by at least two distinct and relatively enduring
+identities or dissociated personalities. These states alternately show in a person's 
+behavior, accompanied by memory impairment for important information not explained 
+by ordinary forgetfulness.
+
+[R: Psychology Today](https://www.psychologytoday.com/conditions/dissociative-identity-disorder-multiple-personality-disorder)
+
+[R: psycom.net](https://www.psycom.net/mchugh.html)
```

Comment:
Please change this ref. The article inside this is poorly written, with unreasonably high usage of words like `more`